bin/check_exercises.sh

```diff
@@ -1,66 +1,45 @@
 #!/usr/bin/env bash
 set -eo pipefail
 
-# can't benchmark with a stable compiler; to bench, use
-# $ BENCHMARK=1 rustup run nightly bin/check_exercises.sh
-if [ -n "$BENCHMARK" ]; then
-    target_dir=benches
-else
-    target_dir=tests
-fi
-
 repo=$(git rev-parse --show-toplevel)
 
 if [ "$GITHUB_EVENT_NAME" = "pull_request" ]; then
     git fetch --depth=1 origin main
     files="$(
         git diff --diff-filter=d --name-only remotes/origin/main |
-        grep "exercises/" || true |
-        cut -d '/' -f -3 |
-        sort -u |
-        awk -v repo="$repo" '{print repo"/"$1}'
```

Comment:
This was an oopsie on my side. The `|| true` belongs at the end of the pipe chain, otherwise the rest of the commands won't run if `grep` actually succeeds.